package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "test": "eslint --fix src && react-scripts test --env=jsdom",
```

Comment:
better not to have the `test` command run `--fix`

Reply:
Removed.